swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md

```diff
@@ -115,64 +121,37 @@ public distinct service class CovidData {
         self.entryRecord = entryRecord.cloneReadOnly();
     }
 
-    resource function get isoCode() returns string {
-        return self.entryRecord.isoCode;
-    }
+    resource function get isoCode() returns string => self.entryRecord.isoCode;
 
-    resource function get country() returns string {
-        return self.entryRecord.country;
-    }
+    resource function get country() returns string => self.entryRecord.country;
 
-    resource function get cases() returns decimal? {
-        if self.entryRecord.cases is decimal {
-            return self.entryRecord.cases / 1000;
-        }
-        return;
-    }
+    resource function get cases() returns int? => self.entryRecord.cases;
 
-    resource function get deaths() returns decimal? {
-        if self.entryRecord.deaths is decimal {
-            return self.entryRecord.deaths / 1000;
-        }
-        return;
-    }
+    resource function get deaths() returns int? => self.entryRecord.deaths;
 
-    resource function get recovered() returns decimal? {
-        if self.entryRecord.recovered is decimal {
-            return self.entryRecord.recovered / 1000;
-        }
-        return;
-    }
+    resource function get recovered() returns int? => self.entryRecord.recovered;
 
-    resource function get active() returns decimal? {
-        if self.entryRecord.active is decimal {
-            return self.entryRecord.active / 1000;
-        }
-        return;
-    }
+    resource function get active() returns int? => self.entryRecord.active;
 }
 ```
 
 In this code:
-- The endpoint returns the number of cases in thousands. Therefore, a service type is used to define the output object type, and inside the service type, each resource method will return the original value
-divided by `1000`.
-- The `CovidData` service type represents the GraphQL `Object` type, which represents an entry in
-the data set. 
-- Each resource method in this service represents a field of the GraphQL object type.  The return type of
+
+* The `CovidData` service type represents the GraphQL `Object` type, which represents an entry in
+the data set.
```

Comment:
Why in a new line?

Reply:
This is according to the [markdown lint rules](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint). It suggests to add a new line before starting a list.